### PR TITLE
OCM-10266 | test: automate ids:75504,57410  Create/Delete cluster and…

### PR DESCRIPTION
**Pull request by yuwang-RH**
[OCM-10266](https://issues.redhat.com//browse/OCM-10266) | test: automate ids:75504,57410  Create/Delete cluster and operator-roles attaching managed policies

logs: https://privatebin.corp.redhat.com/?49ebaaa64d019a32#3CRB4Kv3RkVyFrqYfkhRgiayY2dyHhn8vF3TLge27nKg

### CODE REVIEW

File: tests/e2e/test_rosacli_cluster_post.go [hunk 1]
```diff
@@ -649,4 +650,41 @@ var _ = Describe("Post-Check testing for cluster creation",
...
+					_, operatorRoleName, err := common.ParseRoleARN(policyArn)
+					Expect(err).To(BeNil())
+					roleTags, err := awsClient.IamClient.ListRoleTags(context.TODO(), &iam.ListRoleTagsInput{
+						RoleName: aws.String(operatorRoleName),
+					})
+					Expect(err).To(BeNil())
+					tagCheckPass := false
+					for _, tag := range roleTags.Tags {
+						if *tag.Key == "rosa_managed_policies" && *tag.Value == constants.TrueString {
+							tagCheckPass = true
```
Comment (yasun1) [inline tests/e2e/test_rosacli_cluster_post.go:680]: From the logic, it should `break` if this condition is executed.

File: tests/e2e/test_rosacli_operator_roles.go [hunk 2]
```diff
@@ -946,3 +947,136 @@ var _ = Describe("Detele operator roles with byo oidc", labels.Feature.OperatorR
...
+			err = clusterService.WaitClusterDeleted(clusterID, 3, 30)
+			Expect(err).To(BeNil())
+
+			By("Delete operator-roles")
+			_, err = ocmResourceService.DeleteOperatorRoles(
+				"-c", testingClusterName,
+				"--mode", "auto",
+				"-y",
+			)
+			Expect(err).To(BeNil())
```
Comment (yasun1) [inline tests/e2e/test_rosacli_operator_roles.go:1026]: Is the OIDC Provider be deleted in the function `DeleteCluster`?

**Comment by yuwang-RH** [comment]
@yasun1 Updated in the latest commit, could you help take a look again?

**Pull request merged**